docs/content/2.features/5.seo.md

```diff
@@ -0,0 +1,57 @@
+# SEO 
+
+## General info 
```

Comment:
First paragraph after 1st level heading becomes description. But because of empty space between "SEO" and "General info" this page has no description.

Delete this heading, it is clear that introduction goes right after title.

```suggestion
```